docs/editor/accessibility.md

```diff
@@ -25,6 +25,46 @@ You can adjust the Zoom level in VS Code with the **View** > **Appearance** > **
 
 When you adjust the zoom level with the **View** > **Zoom In / Out** commands, the zoom level is persisted in the `window.zoomLevel` [setting](/docs/getstarted/settings.md). The default value is 0 and each increment/decrement changes the zoom level by 20%.
 
+## Color vision accessibility
+
+You can search for extensions in **Visual Studio Marketplace** that are compatible for color vision deficiency. Use Extensions `kbstyle(Ctrl+Shift+X)` to access the Marketplace extension search bar and enter "colorblind" to populate relevant options.
+
+![Visual Studio Marketplace in VS Code UI](images/accessibility/accessibility-extension-marketplace.png)
+
+To change the [color theme](/docs/getstarted/themes.md), go to **File** > **Preferences** > **Color Theme** `kbstyle(Ctrl+K)` `kbstyle(Ctrl+T)`.
+
+![Dropdown for Select Color Theme](images/accessibility/accessibility-select-theme.png)
+
+### Customizing warning colors in VS Code
+
+The default Color Theme for VS Code is **Dark+**. However, you can customize both the theme and property colors in the user interface.
+
+> Note: Visit [Customizing a Color Theme](/docs/getstarted/themes.md#_customizing-a-color-theme) to learn more about overriding the colors in your current theme.
+
+To customize the error/warning squigglies, go to **File** > **Preference** > **Settings**. Then open the UI settings by selecting the file icon shown in the image below.
+
+![JSON file settings icon](images/accessibility/accessibility-settings-json-file.png)
+
+Once the `kbstyle(settings.json)` file populates, nest the following code inside the outermost curly braces. You'll be able to assign a color to each object by entering a hex code.
+
+```JSON
+    "workbench.colorCustomizations": {
+
+    "editorError.foreground": "#ffef0f",
```

Comment:
Are these color especially good for people with color blindness? Should we mention what they are and why these colors were chosen?

Reply:
Thanks for pointing this out! I added a line under this section, explaining the reason for this. I also included the different types of color blindness.